bindings/python/README.md

```diff
@@ -34,33 +34,14 @@ asyncio.run(main())
 
 ## Development
 
-Setup virtualenv:
-
-```shell
-python -m venv venv
-```
-
-Activate venv:
-
-```shell
-source venv/bin/activate
-````
-
-Install `maturin`:
-
-```shell
-pip install maturin[patchelf]
-```
-
 Build bindings:
 
 ```shell
-maturin develop
+just build_bindings_python
```

Comment:
This kind of integration is very confusing.
`just` will not prepare the environment for developers.

Reply:
+1

Reply:
I noticed that Xuanwo has added the part about environment preparation in justfile

Reply:
It doesn't seem complete. 
I don't think it is suitable for us to handle these, and our environment preparation is not very complex.